app/scripts/modules/core/src/pipeline/config/stages/overrideFailure/overrideFailure.component.js

```diff
@@ -12,37 +12,11 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.overrideFailure',
       this.viewState = {};
 
       this.failureOptionChanged = () => {
-        if (this.viewState.failureOption === 'fail') {
-          this.stage.failPipeline = true;
-          this.stage.continuePipeline = false;
-          this.stage.completeOtherBranchesThenFail = false;
-        } else if (this.viewState.failureOption === 'stop') {
-          this.stage.failPipeline = false;
-          this.stage.continuePipeline = false;
-          this.stage.completeOtherBranchesThenFail = false;
-        } else if (this.viewState.failureOption === 'ignore') {
-          this.stage.failPipeline = false;
-          this.stage.continuePipeline = true;
-          this.stage.completeOtherBranchesThenFail = false;
-        } else if (this.viewState.failureOption === 'faileventual') {
-          this.stage.failPipeline = false;
-          this.stage.continuePipeline = false;
-          this.stage.completeOtherBranchesThenFail = true;
-        }
+        this.stage.onFailure = this.viewState.failureOption;
```

Comment:
Not sure if this is right or whether it can be simplified even more by just directly binding to `this.stage.onFailure`.